Android.mk

```diff
@@ -277,7 +282,14 @@ endif
 endif
 include $(BUILD_EXECUTABLE)
 
+###############################################################
+# build minisfservice executable
 include $(CLEAR_VARS)
+
+LOCAL_CLANG := true
+
+LOCAL_C_INCLUDES := frameworks/native/services/surfaceflinger
```

Comment:
The LOCAL_C_INCLUDES modified should be inside the section specific to the hack.